charts/consul/values.yaml

```diff
@@ -263,6 +275,44 @@ global:
         additionalConfig: |
           {}
 
+      controller:
+        # The directory that Kubernetes will use on Kubernetes CRD creation, 
+        # deletion, and update, to get TLS certificates used issued from vault
+        # to send webhooks to the controller.
+        tlsCert:
+          # The Vault secret path that issues TLS certificates for controller
+          # webhooks.
+          # @type: string
+          secretName: null
+          
+        # The directory that Kubernetes will use on Kubernetes CRD creation, 
+        # deletion, and update, to get CA certificates used issued from vault
+        # to send webhooks to the controller.
+        caCert:
+          # The Vault secret path that contains the CA certificate for controller
+          # webhooks.
+          # @type: string
+          secretName: null
+      
+      connectInject:
+        # The directory that Kubernetes will use on Kubernetes CRD creation, 
+        # deletion, and update, to get CA certificates used issued from vault
+        # to send webhooks to ConnectInject
+        caCert: 
+          # The Vault secret path that contains the CA certificate for
+          # Connect Inject webhooks.
+          # @type: string
+          secretName: null
+        
+        # The directory that Kubernetes will use on Kubernetes CRD creation, 
+        # deletion, and update, to get TLS certificates used issued from vault
+        # to send webhooks to Connect Inject
```

Comment:
This description also doesn't seem accurate. This value is for the secret, correct?

Reply:
done